cmd/resource-topology-exporter/main.go

```diff
@@ -219,27 +220,33 @@ func argsParse(argv []string) (nrtupdater.Args, resourcemonitor.Args, resourceto
 		}
 		resourcemonitorArgs.ExcludeList.ExcludeList = conf.ExcludeList
 		localArgs.SysConf = conf.Resources
+		// do not overwrite with empty an existing value (e.g. from opts)
+		if rteArgs.TopologyManagerPolicy == "" {
+			rteArgs.TopologyManagerPolicy = conf.TopologyManagerPolicy
+		}
 	}
+
 	if tmPolicy, ok := arguments["--topology-manager-policy"].(string); ok {
-		if tmPolicy == "" {
-			// last attempt
-			tmPolicy = os.Getenv("TOPOLOGY_MANAGER_POLICY")
+		// empty string is a valid value here, so just keep going;
+		// but not overwrite with empty an existing value (e.g. from conf)
+		if rteArgs.TopologyManagerPolicy == "" {
+			rteArgs.TopologyManagerPolicy = tmPolicy
 		}
-		// empty string is a valid value here, so just keep going
-		rteArgs.TopologyManagerPolicy = tmPolicy
 	}
 
 	return nrtupdaterArgs, resourcemonitorArgs, rteArgs, localArgs, nil
 }
 
 type config struct {
```

Comment:
This struct getting bigger and it would be nice to have it outside of the ```main.go```.
At least one good reason that I can think of is that we would like to get rid of docopt and can use the ```config``` to initialize some of the args.
for example:
```
func (re *ResourceExcludeList) Set(value string) error {
            config, err := ReadConfig(value)
            re.ExcludeList = config.ExcludeList
    }
```

Reply:
fixed